iot-e2e-tests/common/src/test/java/tests/integration/com/microsoft/azure/sdk/iot/iothub/methods/DeviceMethodTests.java

```diff
@@ -128,74 +128,6 @@ public void serviceClientTokenRenewalWithAzureSasCredential() throws Exception
         super.invokeMethodSucceed();
     }
 
-    @Test
-    @StandardTierHubOnlyTest
-    @ContinuousIntegrationTest
-    public void invokeMethodInvokeParallelSucceed() throws Exception
-    {
-        // Arrange
-        super.openDeviceClientAndSubscribeToMethods();
-        CountDownLatch cdl = new CountDownLatch(NUMBER_INVOKES_PARALLEL);
-        List<RunnableInvoke> runs = new LinkedList<>();
-
-        for (int i = 0; i < NUMBER_INVOKES_PARALLEL; i++)
-        {
-            RunnableInvoke runnableInvoke;
-            // Create one methodServiceClient per thread since each method service client only allows one method invoke
-            // at a time. This limitation exists because the invokeMethod method is synchronized with itself
-            DeviceMethod methodServiceClient = DeviceMethod.createFromConnectionString(iotHubConnectionString, DeviceMethodClientOptions.builder().httpReadTimeout(HTTP_READ_TIMEOUT).build());
-            if (testInstance.identity instanceof TestModuleIdentity)
-            {
-                runnableInvoke = new RunnableInvoke(methodServiceClient, testInstance.identity.getDeviceId(), ((TestModuleIdentity) testInstance.identity).getModuleId(),"Thread" + i, cdl);
-            }
-            else
-            {
-                runnableInvoke = new RunnableInvoke(methodServiceClient, testInstance.identity.getDeviceId(), null,"Thread" + i, cdl);
-            }
-            new Thread(runnableInvoke).start();
-            runs.add(runnableInvoke);
-        }
-
-        cdl.await(3, TimeUnit.MINUTES);
-
-        for (RunnableInvoke run:runs)
-        {
-            MethodResult result = run.getResult();
-            assertNotNull(buildExceptionMessage(run.getException() == null ? "Runnable returns null without exception information" : run.getException().getMessage(), testInstance.deviceTestManager.client), result);
-            assertEquals(buildExceptionMessage("Expected SUCCESS but was " + result.getStatus(), testInstance.deviceTestManager.client), (long) DeviceEmulator.METHOD_SUCCESS,(long)result.getStatus());
-            assertEquals(buildExceptionMessage("Expected payload " + run.getExpectedPayload() + " but got " + result.getPayload().toString(), testInstance.deviceTestManager.client), run.getExpectedPayload(), result.getPayload().toString());
-        }
-    }
-
-    @Test
-    @StandardTierHubOnlyTest
-    @ContinuousIntegrationTest
-    public void invokeMethodStandardTimeoutSucceed() throws Exception
```

Comment:
Too similar to the basic invokeMethod test. The only difference is that the default timeout values are used. There are already unit tests in place that make sure that the SDK has default values for this scenario